recipes/r-spatstat/meta.yaml

```diff
@@ -0,0 +1,107 @@
+{% set version = '1.53-2' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-spatstat
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: spatstat_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/spatstat_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/spatstat/spatstat_{{ version }}.tar.gz
+  sha256: 0bc6d923e1fa59a87c3c9364d3d72aff5502db57dddf23aab78f01442e4c65ad
+
+build:
+  number: 0
+  script: R CMD INSTALL --build .
+  skip: true  # [win32]
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - r-matrix
+    - r-abind
+    - r-deldir >=0.0_21
+    - r-goftest
+    - r-mgcv
+    - r-nlme
+    - r-polyclip >=1.5_0
+    - r-rpart
+    - r-spatstat.data >=1.1_0
+    - r-spatstat.utils >=1.7_1
```

Comment:
r-spatstat.data and r-spatstat.utils need to be added to conda-forge before this recipe can be built. @jsignell could you please send separate PRs for these two recipes?